test/Microsoft.IdentityModel.Tokens.Tests/AuthenticatedEncryptionProviderTests.cs

```diff
@@ -88,6 +87,44 @@ public override string ToString()
     /// </summary>
     public class AuthenticatedEncryptionProviderTests
     {
+#if NET_CORE
```

Comment:
nit: you could merge these two tests together and use
`[PlatformSpecific(TestPlatforms.OSX | TestPlatforms.Linux)]`